src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -43,6 +44,10 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
+### Key Transactions
+
+Typically, for most projects there are only a few transactions that you really care about. We prioritize key, or starred, transactions over others, so you can have more visibility into your important data.
```

Comment:
"we prioritize key" is not the same as "starred"? I feel like we could like the section where we explain what is a starred transaction here

Reply:
AFAIK "starred" means -> 
![image](https://user-images.githubusercontent.com/1374633/203602579-89bf92f5-77d3-4801-9ebc-8760f501cac7.png)

Reply:
yes  "starred" is clear to me but what does mean "we prioritized by _key **or** starred_"?

Reply:
@ale-cota can you please help here?

Reply:
I think key transactions is the internal term, whereas starred is the external one.